mfrc522/iso14443com.py

```diff
@@ -0,0 +1,55 @@
+import crcmod
+from .mfrc522 import *
+import struct
+
+SEL_CASCADE_1   = 0x93
+SEL_CASCADE_2   = 0x95
+SEL_CASCADE_3   = 0x97
+
+__crc_func = crcmod.mkCrcFun(0x11021, initCrc=0x6363)
+
+def __calculate_crc(data):
+    return struct.pack('<H', __crc_func(data))
+
+def __perform_cascade(module, cascade_level):
+    if cascade_level != SEL_CASCADE_1 and \
```

Comment:
Preferably rewrite this `if` to:
```python
if cascade_level not in [ SEL_CASCADE_1, SEL_CASCADE_2, SEL_CASCADE_3 ]
```

...or even better make the enum and say `if cascade_level not in <enum>`.